modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -280,6 +280,12 @@ public void postProcessParameter(CodegenParameter parameter) {
                 }
             }
 
+            // If httpClient is used and 204 possible, then null might be returned. In this case we declare it for proxy return type.
+            boolean responseCanBeNull = (boolean) additionalProperties.get("useHttpClient") && op.responses.anyMatch(r -> r.code.equals("204"));
```

Comment:
I am not sure we're setting the property `useHttpClient`, I think I changed the templates to simply always use it and ignore that property, so maybe you want to do the same here?

Reply:
@pgrm `additionalProperties.put("useHttpClient", ngVersion.atLeast("4.3.0"));`